packages/s/snapd/files/wrapper.sh

```diff
@@ -29,15 +29,18 @@ fi
 
 if [[ "${CONFINEMENT}" != "strict" ]] && [[ "${DISABLE_CONFINEMENT_WARNING:-n}" != "y" ]]
   then
-  if [[ -n "${BAMF_DESKTOP_FILE_HINT+x}" ]] && [[ -n "${GIO_LAUNCHED_DESKTOP_FILE+x}" ]]
+  # This prevents the warning from showing on KDE Plasma
+  # if [[ -n "${BAMF_DESKTOP_FILE_HINT+x}" ]] && [[ -n "${GIO_LAUNCHED_DESKTOP_FILE+x}" ]]
+
+  if [[ -n "${BAMF_DESKTOP_FILE_HINT+x}" ]]
   then
       notify-send \
           --app-name Snap \
-          --urgency normal \
+          --urgency critical \
           --icon dialog-warning \
           "Snap has ${CONFINEMENT} confinement" \
-          "Snaps will stop working in early January 2025." \
-          "See ${URL} for details."
+          "Snaps will stop working in early January 2025. For details see<br> \
+          ${URL}"
```

Comment:
This doesn't work for me:

![image](https://github.com/user-attachments/assets/fa61f0ac-adcd-4580-b21a-15eecce9a4f1)

It also overflows the regular notification:

![image](https://github.com/user-attachments/assets/8882a5ac-436d-4f2a-93b7-c8e5aec219ca)

Maybe trim it down so at least `help.getsol.us` is visible:

```suggestion
          "Snaps will stop working in early January 2025. See ${URL}"
```

Reply:
Well, that lack of ability to translate the br tag is disappointing. My bad for not testing on the other DEs.
I've trimmed down the message and removed the tag.
I tested on Plasma, Budgie and gnome. It looks good on those.
Sadly, my XFCE VM seems to have died after updates.

If you still see an overflow, let me know what DE it's in.